@xen-orchestra/lite/src/components/pool/network/PoolHostInternalNetworksTable.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+  <div class="pool-host-internal-networks-table">
+    <UiTitle>
+      {{ $t('host-internal-networks') }}
+      <template #actions>
+        <UiButton
+          v-tooltip="$t('coming-soon')"
+          disabled
+          :left-icon="faPlus"
+          variant="secondary"
+          accent="info"
+          size="medium"
+        >
+          {{ $t('new') }}
+        </UiButton>
+      </template>
+    </UiTitle>
+    <div class="container">
+      <div class="table-actions">
+        <UiQuerySearchBar @search="value => (searchQuery = value)" />
+        <UiTableActions :title="t('table-actions')">
+          <UiButton
+            v-tooltip="$t('coming-soon')"
+            disabled
+            :left-icon="faEdit"
+            variant="tertiary"
+            accent="info"
+            size="medium"
+          >
+            {{ $t('edit') }}
+          </UiButton>
+          <UiButton
+            v-tooltip="$t('coming-soon')"
+            disabled
+            :left-icon="faTrash"
+            variant="tertiary"
+            accent="danger"
+            size="medium"
+          >
+            {{ $t('delete') }}
+          </UiButton>
+        </UiTableActions>
+        <UiTopBottomTable
+          :selected-items="selected.length"
+          :total-items="networkUuids.length"
+          @toggle-select-all="toggleSelect"
+        />
+      </div>
+      <VtsDataTable
+        :is-ready
+        :has-error
+        :no-data-message="networks.length === 0 ? $t('no-network-detected') : undefined"
+      >
+        <template #thead>
+          <tr>
+            <template v-for="column of visibleColumns" :key="column.id">
+              <th v-if="column.id === 'checkbox'" class="checkbox">
+                <UiCheckbox :v-model="areAllSelected" accent="info" @update:model-value="toggleSelect" />
+              </th>
+              <th v-else-if="column.id === 'more'" v-tooltip="$t('coming-soon')" class="more">
+                <UiButtonIcon size="small" accent="info" :icon="faEllipsis" />
+                {{ column.label }}
+              </th>
+              <ColumnTitle v-else :icon="headerIcon[column.id]">
+                <span class="text-ellipsis">{{ column.label }}</span>
+              </ColumnTitle>
```

Comment:
This component is destined to be replaced.

Also, it makes the ellipsis not working.

I suggest using a simple `th` instead for now.

```suggestion
              <th v-else>
                <div v-tooltip class="text-ellipsis">
                  <VtsIcon accent="brand" :icon="headerIcon[column.id]" />
                  {{ column.label }}
                </div>
              </th>
```